pkg/client/client.go

```diff
@@ -1075,3 +1096,40 @@ func DoesLocalBucketExist(serverURL string, bucket string) (bool, error) {
 
 	return false, nil
 }
+
+func WaitMapVersionSync(timeout time.Time, smap dfc.Smap, idsToIgnore []string) error {
+	prev := smap.Version
+	urls := make(map[string]struct{})
+	for _, d := range smap.Tmap {
+		for _, ignore := range idsToIgnore {
```

Comment:
when I was fed up with writing similar loops I added a simple function to regression_test: stringInSlice :) 
It returns bool, so you can replace it with if !stringInSlice(d.DaemonID, idsToIgnore) { urls[....
But if you keep loops, it is fine to me

Reply:
Thanks for the heads up! I think I'll keep this loop as-is for now (since `stringInSlice()` is in a different package), but the next time I have to do something like this, I'll add a generic function to `utils.go` (or somewhere that makes sense).

Reply:
Yes, I agree. I missed that it is in different packages. But util.go function may be helpful next time.